pkg/controller.v1/pytorch/envvar.go

```diff
@@ -25,6 +25,17 @@ import (
 	kubeflowv1 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
 )
 
+const (
+	// Worker/node size related arguments.
+
+	// EnvNprocPerNode is the environment variable name for the number of processes per node.
+	EnvNprocPerNode = "PET_NPROC_PER_NODE"
```

Comment:
So, PET_NPROC_PER_NODE is used in non elastic mode and NProcPerNode inside elasticPolicy is used in elastic mode serving the same purpose. Is it right? If then, can we add a validation check to avoid setting PET_NPROC_PER_NODE when elastic mode is enabled

Reply:
Yes, they are the same and I prefer cleanup the one inside elasticPolicy in the future, since it more flexible and it was mispelled.

Anyway, nproc_per_node has nothing to do with elastic.